x509-tsp/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "x509-tsp"
+version = "0.1.0"
+edition = "2021"
+description = """
+Pure Rust implementation of the Timestamping Protocol as described in RFC 3161.
+"""
+authors = ["RustCrypto Developers"]
+license = "Apache-2.0 OR MIT"
+repository = "https://github.com/RustCrypto/formats/tree/master/cms"
```

Comment:
Looks like this should be renamed to `https://github.com/RustCrypto/formats/tree/master/x509-tsp`